guides/common/modules/proc_upgrading-your-project-server.adoc

```diff
@@ -23,7 +23,7 @@ ifdef::katello,satellite,orcharhino[]
 +
 [options="nowrap" subs="attributes"]
 ----
-$ podman login registry.redhat.io
+$ podman login --authfile /etc/foreman/registry-auth.json registry.redhat.io
```

Comment:
The prefix for this command implies it runs as a normal user but I'd expect it to run as root